crypto/engine/eng_dyn.c

```diff
@@ -184,10 +184,12 @@ static int dynamic_set_data_ctx(ENGINE *e, dynamic_data_ctx **ctx)
         }
     }
     CRYPTO_THREAD_unlock(global_engine_lock);
+    ret = 1;
     /*
      * If we lost the race to set the context, c is non-NULL and *ctx is the
      * context of the thread that won.
      */
+end:
     if (c)
```

Comment:
Even though it's preexisting, could the `!= NULL` be added please.

Reply:
> Even though it's preexisting, could the `!= NULL` be added please.

done.